package.json

```diff
@@ -12,32 +12,33 @@
   "repository": "",
   "scripts": {
     "build": "ember build",
-    "start": "ember server",
+    "start": "ember serve",
```

Comment:
when did this change?

Reply:
Looks like we don't have 1 consistent command. CLI docs say server, and various parts of the guides say serve

Reply:
it changed bc the default blueprint changed